api/institutions/serializers.py

```diff
@@ -359,6 +359,53 @@ def get_absolute_url(self):
         return None  # there is no detail view for institution-users
 
 
+class NewInstitutionSummaryMetricsSerializer(JSONAPISerializer):
+    '''serializer for institution-summary metrics
+
+    used only when the INSTITUTIONAL_DASHBOARD_2024 feature flag is active
+    (and should be renamed without "New" when that flag is permanently active)
+
+    Summary contains counts of
+    - Total users in the institution
+    - Total public project count for the institution
+    - Total private project count for the institution
+    - Total public registration count for the institution
+    - Total private registration count for the institution
+    - Total published preprint count for the institution
+
```

Comment:
nit: i'd say this kind of info is not helpful in the docstring, since it duplicates info in the fields themselves and is unlikely to be kept in sync as fields are added/removed over time (again, thinking of confusion for future contributors) -- and here it's already out of sync (missing the last four fields)